cli/copy.c

```diff
@@ -87,7 +87,7 @@ j_cmd_copy (gchar const** arguments)
 			}
 			else if (i == 1)
 			{
-				g_autoptr(JItem) item = NULL;
+				G_GNUC_UNUSED g_autoptr(JItem) item = NULL;
```

Comment:
Why? It is used later on. Does Clang warn about it?

Reply:
item = j_item_create ...
Is an assignment, but the variable is never read.
The used static code analyses warns me about this.